src/helpers/polyfill.ts

```diff
@@ -0,0 +1,3 @@
+if (!(global as any)._babelPolyfill) {
+  require("babel-polyfill");
```

Comment:
Have you tried surrounding the require in a `try/catch` block? If this trick works, it might cover more versions and avoids touching internals.